packages/react-aria-components/test/Disclosure.test.js

```diff
@@ -296,6 +296,37 @@ describe('Disclosure', () => {
     await user.click(trigger2);
     expect(panel2).not.toBeVisible();
   });
+
+  it('should not expand or collapse on repeat keydown events', async () => {
+    let onExpandedChange = jest.fn();
+    const {getByRole, queryByText} = render(
+      <Disclosure onExpandedChange={onExpandedChange}>
+        <Heading level={3}>
+          <Button slot="trigger">Trigger</Button>
+        </Heading>
+        <DisclosurePanel>
+          <p>Content</p>
+        </DisclosurePanel>
+      </Disclosure>
+    );
+
+    const panel = queryByText('Content');
+    const button = getByRole('button');
+
+    await user.tab();
+
+    expect(panel).not.toBeVisible();
+    expect(button).toHaveFocus();
+
+    // press Enter without releasing it and trigger 4 keydown events
+    await user.keyboard('{Enter>4}');
+    expect(panel).toBeVisible();
+    expect(onExpandedChange).toHaveBeenCalledTimes(1);
+
+    await user.keyboard('{Enter>4}');
+    expect(panel).not.toBeVisible();
+    expect(onExpandedChange).toHaveBeenCalledTimes(2);
```

Comment:
make sure to end the user keyboard event
just in case the `user` state carries over into other tests